online_meetings/canvas_zoom_meetings.py

```diff
@@ -201,12 +203,23 @@ def main() -> Sequence[Dict[str, Union[ValidDataSourceName, pd.Timestamp]]]:
     )
 
     zoom_courses_df = pd.DataFrame(zoom_placements.zoom_courses)
-    zoom_courses_df.index.name = "id"
+    zoom_courses_df = zoom_courses_df.set_index(["id", "placement_type_id"])
+
     zoom_courses_meetings_df = pd.DataFrame(zoom_placements.zoom_courses_meetings)
     zoom_courses_meetings_df.index.name = "id"
 
     zoom_courses_df.to_csv(os.path.join(DATA_DIR, "zoom_courses.csv"))
     zoom_courses_meetings_df.to_csv(os.path.join(DATA_DIR, "zoom_courses_meetings.csv"))
```

Comment:
I'd like to suggest nesting this under an `if CREATE_CSVS`, like so:

https://github.com/tl-its-umich-edu/course-inventory/blob/163704da5037b81998e858f9c23225ba8fe7a9e0/course_inventory/inventory.py#L330-L334

Reply:
Okay, didn't notice that value, thanks!